packages/sdk/tests/common/http/FetchHttpClient.logging.unit.test.ts

```
@@ -39,14 +39,22 @@ describe('FetchHttpClient logging', () => {
-            message: 'HTTP Response',
-            source: 'FetchHttpClient'
-        });
-        expect(callArgs.context).toBeDefined();
+        try {
+            await httpClient.get();
+            throw new Error('Should not reach here');
+        } catch (error) {
+            expect(error).toBeInstanceOf(HttpException);
+            const httpError = error as HttpException;
+            expect(httpError.status).toBe(403);
+            
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)